### ISSUE TITLE
[BUG] [Java Service]Unclear message is returned when requesting existing key without source parameter in string-based GET API.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1452]
**Describe the bug**
commit: 0531224d52a515ce3da7f72560ddd899628f03e6
Unclear message is returned when requesting existing key without source parameter in string-based GET API.
It occurs in v1 and v2 API.

**To Reproduce**
1. Go to get a source API:
/i18n/api/v2/translation/products/{productName}/versions/{version}/locales/{locale}/components/{component}/keys/{key}
2. Add below parameters to API request: (make sure requested product(i.e.. VMwareVIP2018) is in product supported list)
https://localhost:8090/i18n/api/v2/translation/products/VMwareVIP2018/versions/1.1.0/locales/zh-hans/components/default/keys/Partner_Name?collectSource=true&pseudo=false
3. Get the response as below:
data/status: The translation is found and returned, but the according en string is not matched to source.
```
{
  "response": {
    "code": 200,
    "message": "OK",
    "serverTime": ""
  },
  "signature": "",
  "data": {
    "productName": "VMwareVIP2018",
    "version": "1.1.0",
    "dataOrigin": "",
    "pseudo": false,
    "machineTranslation": false,
    "source": null,
    "translation": "合作伙伴名称",
    "locale": "zh-Hans",
    "key": "Partner_Name",
    "component": "default",
    "status": "The translation is found and returned, but the according en string is not matched to source."
  }
}
```

**Expected behavior**
The message of /data/status should be more clear.
i.e. Source is not provided, the existing translation is found and returned.

**Comment by InaSong** [comment]
Assign to Tiger for first round of triage.

**Comment by lyiyu66** [comment]
The issue is fixed with commit 93ffa26a7c0616ff7fee3eb25318342d1dc8975b
